packages/react-select/src/components/containers.tsx

```diff
@@ -85,9 +85,10 @@ export const valueContainerCSS = <
   Group extends GroupBase<Option>
 >({
   theme: { spacing },
+  isMulti
 }: ValueContainerProps<Option, IsMulti, Group>): CSSObjectWithLabel => ({
   alignItems: 'center',
-  display: 'flex',
+  display: `${isMulti ? 'flex' : 'grid'}`,
```

Comment:
Question of style, not critical, but why literals when you can do it without them?